react/src/components/atoms/forms/Input/index.js

```diff
@@ -36,19 +40,72 @@ class InputProvider extends React.Component {
     super(props);
     this.state = {
       value: this.props.defaultValue,
+      linkedContent: this.props.linkedContent,
```

Comment:
can you add propTypes for Input and document all the props that Input/InputProvider expect?